src/app/oppdateringslogg/OppdateringsloggContainer.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import ModalWrapper from 'nav-frontend-modal';
+import useListener from '../../utils/hooks/use-listener';
+import Oppdateringslogg from './Oppdateringslogg';
+import { Systemtittel } from 'nav-frontend-typografi';
+import styled from 'styled-components/macro';
+import { OppdateringsloggType } from './EnkeltOppdateringslogg';
+import usePersistentState from './usePersistentState';
+import useOppdateringslogg from './useOppdateringslogg';
+import './oppdateringsloggKnapp.less';
+import useWaitForElement from '../../utils/hooks/use-wait-for-element';
+
+export const DecoratorButtonId = 'oppdateringslogg';
+export interface EnOppdateringslogg {
+    id: number;
+    tittel: string;
+    dato: Date;
+    aktiv: boolean;
+    ingress: string;
+    beskrivelse: string;
+    type: OppdateringsloggType;
```

Comment:
Ser ikke ut som vi bruker type lengre og da kan vi ta den vekk kanskje? evt legge den tilbake dersom det blir behov for det senere

Reply:
Fjernet `type` fra alle filene hvor det ble brukt, samt `OppdaterubgsloggType`.